src/js/components/Calendar/Calendar.js

```diff
@@ -847,21 +846,24 @@ const Calendar = forwardRef(
         if (!children) {
           days.push(
             <CalendarDay
+              activeProp={active}
+              day={day}
               key={day.getTime()}
+              disabledProp={dayDisabled && !!dayDisabled}
               buttonProps={{
                 a11yTitle: day.toDateString(),
-                active: active && active.getTime() === day.getTime(),
-                disabled: dayDisabled && !!dayDisabled,
-                onClick: () => onClick(dateObject),
-                onMouseOver: () => setActive(dateObject),
-                onMouseOut: () => setActive(undefined),
+                onClick:
+                  dayDisabled && !!dayDisabled
+                    ? () => {}
+                    : () => onClick(dateObject),
               }}
               isInRange={inRange}
               isSelected={selected}
               otherMonth={day.getMonth() !== reference.getMonth()}
               rangePosition={rangePosition}
               size={size}
               fill={fill}
+              ref={focusableDateRef}
```

Comment:
you aren't tracking custom days so does that mean they don't get the same navigation?

Reply:
Yeah I didn't go into custom dates for this PR. Seems like the customDates needs a decent amount of work o held off for now. If a calendar is build with customDates it is all tab based navigation.